src/ui/reducers/app.ts

```diff
@@ -194,6 +199,8 @@ export default appSlice.reducer;
 const getSelectedPanel = (state: UIState) => state.layout.selectedPanel;
 const getViewMode = (state: UIState) => state.layout.viewMode;
 
+export const getReplayAssist = (state: UIState) => state.app.showReplayAssist;
```

Comment:
Similar nit about "get blah" vs "show blah" – these names should probably better match.

Reply:
After that variable rename– there is a naming conflict. (You have two things named `showReplayAssist`).

One way to fix that is with an import alias:
```ts
import { showReplayAssist as showReplayAssistSelector } from "ui/reducers/app";
```

And then down in the component bodyL
```ts
const showReplayAssist = useAppSelector(showReplayAssistSelector);
```

That being said, maybe it's easier to ignore this and I'll do a small (sort of subjective) naming sweep myself.